docs/settings.md

```diff
@@ -252,6 +252,37 @@ HTTP/1.1 204 No Content
 
 ## Instance
 
+### GET /settings/capabilities
+
+List the activated capabilities for this instance.
```

Comment:
Nit:
*An unadvertised capability should be considered `false` and, for backward comaptibility, if you can't get a valid response from this endpoint (in particular in case of a `404 Not found` error), all capabilities should be considered `false`*